lib/screens_web/controllers/v2/screen_controller.ex

```diff
@@ -128,17 +128,17 @@ defmodule ScreensWeb.V2.ScreenController do
 
     [
       app_id: app_id,
-      refresh_rate: refresh_rate,
-      audio_readout_interval: Parameters.audio_interval_minutes(config),
       audio_interval_offset_seconds: Parameters.audio_interval_offset_seconds(config),
-      sentry_dsn: if(params["disable_sentry"], do: nil, else: Sentry.get_dsn()),
-      refresh_rate_offset: calculate_refresh_rate_offset(screen_id, refresh_rate),
+      audio_readout_interval: Parameters.audio_interval_minutes(config),
+      is_pending: false,
       is_real_screen: match?(%{"is_real_screen" => "true"}, params),
-      screen_side: screen_side(params),
+      refresh_rate_offset: calculate_refresh_rate_offset(screen_id, refresh_rate),
+      refresh_rate: refresh_rate,
       requestor: params["requestor"],
       rotation_index: rotation_index(params),
-      variant: params["variant"],
-      is_pending: false
+      screen_side: screen_side(params),
+      sentry_dsn: if(params["disable_sentry"], do: nil, else: Sentry.get_dsn()),
+      variant: params["variant"]
```

Comment:
I do love a bit of alphabetization, but after the refactor this file is otherwise untouched by the changes, so maybe we should leave it be until we have some other reason to change it? Or it could be a separate refactor commit.